dts/bindings/ethernet/intel,e1000.yaml

```diff
@@ -13,3 +13,8 @@ properties:
 
     interrupts:
       required: true
+
+    pcie:
```

Comment:
Is there some other way the e1000 can be attached other than PCIe?

Reply:
Driver only supports PCIe so no.

Reply:
Then I don't see any purpose in having this as a DTS property.

Reply:
If I remove this from yaml, then the cmake stage will fail.